pandas/tests/window/test_api.py

```diff
@@ -342,3 +342,32 @@ def test_multiple_agg_funcs(self, func, window_size, expected_vals):
         )
 
         tm.assert_frame_equal(result, expected)
+
+
+@td.skip_if_no("numba", "0.46.0")
```

Comment:
can you use engine fixture?

Reply:
Moved to a specific test where it only make sense with numba installed.